### ISSUE TITLE
I want to know how to adjust the speed of the sentence synthesized by tacotron2 + MB melgan. He speaks too fast, but I don't know where to adjust the speed and timbre

### ISSUE THREAD

**Issue by yukikazenanodesu** [TensorSpeech/TensorFlowTTS#631]
here is my result  
[result.zip](https://github.com/TensorSpeech/TensorFlowTTS/files/6882332/result.zip)

**Comment by yukikazenanodesu** [comment]
and one more question is that How does he deal with thousands of words of long sentences while maintaining good results

**Comment by OnceJune** [comment]
you might need to add prosody label to your training data and synth text

**Comment by yukikazenanodesu** [comment]
thanks
